subsys/bluetooth/host/hci_core.c

```diff
@@ -3528,6 +3584,9 @@ static const struct event_handler normal_events[] = {
 #if defined(CONFIG_BT_CONN)
 	EVENT_HANDLER(BT_HCI_EVT_DISCONN_COMPLETE, hci_disconn_complete,
 		      sizeof(struct bt_hci_evt_disconn_complete)),
+	EVENT_HANDLER(BT_HCI_EVT_REMOTE_VERSION_INFO,
+			  bt_hci_evt_read_remote_version_complete,
+			  sizeof(struct bt_hci_evt_remote_version_info)),
```

Comment:
The split lines don't seem to be properly aligned with the opening parenthesis of the first line

Reply:
This is a really strange error, because in my IDE (vscode) all of the event handlers are not aligned properly and my addition is aligned with the others.
Apparently vscode and github has different lengths of tabs?
The alignment on my addition was three tabs and two spaces while the alignment of the others are two tabs and six spaces.
For now I just fixed mine to match the others although there is no visual difference on my side (in my IDE).